pkgs/development/python-modules/ansible-gendoc/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, installShellFiles
+, importlib-metadata
+, jinja2
+, rich
+, pyaml
+, gitpython
+, typer
+, giturlparse
+}:
+
+buildPythonPackage rec {
+  pname = "ansible-gendoc";
+  version = "0.1.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "tZj40TzmhPUPANCL48Jsmeyzv8PVFHtsSdKQPkS0b+k=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  propagatedBuildInputs = [
+    importlib-metadata
+    jinja2
+    rich
+    pyaml
+    gitpython
+    typer
+    giturlparse
+  ];
+
+  doCheck = false;
+
+  postInstall = ''
```

Comment:
See https://github.com/NixOS/nixpkgs/issues/308283

Reply:
Thanks a lot for your feedback, unfortunately I tried to specify the build system and it appears that this package rely on old version of other packages that aren't in nixpkgs and `pythonRelaxDeps` does not seem to work, I think I'll have to close the PR :/ but thanks for taking the time to review this :)